src/pybind/mgr/dashboard/frontend/src/app/shared/services/combo-box.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ComboBoxService } from './combo-box.service';
+
+describe('ComboBoxService', () => {
+  let service: ComboBoxService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(ComboBoxService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
```

Comment:
We should have some tests here

Reply:
tests are added to the directive itself.